prac_05/word_occurrences.py

```diff
@@ -0,0 +1,11 @@
+word_to_occurrence = {}
+text = input("Text: ")
+words = text.split()
+for word in words:
+    frequency = word_to_occurrence.get(word, 0)
+    word_to_occurrence[word] = frequency + 1
```

Comment:
This works, however, you could make this a one liner like Lindsays prac video:
word_to_occurrence[word] = word_to_occurrence.get(word, 0) + 1

Reply:
list comprehension - good find!